projects/1.19/assets/create-planetary-tweaks/cp_tweaks/lang/zh_cn.json

```
@@ -0,0 +1,80 @@
+{
+   
+  "item.cp_tweaks.burning": "燃烧图标",
+  "item.cp_tweaks.warm": "温暖图标",
+  "item.cp_tweaks.comfy": "舒适图标",
+  "item.cp_tweaks.cold": "寒冷图标",
+  "item.cp_tweaks.freezing": "冰冻图标",
+  
+  "item.cp_tweaks.autumn": "秋季图标",
+  "item.cp_tweaks.dry": "干燥图标",
+  "item.cp_tweaks.spring": "春季图标",
+  "item.cp_tweaks.summer": "夏季图标",
+  "item.cp_tweaks.wet": "潮湿图标",
+  "item.cp_tweaks.winter": "冬季图标",
+  
+  "item.cp_tweaks.void_worm_gu": "虚空蠕虫黏液",
+  "item.cp_tweaks.steel_ingot": "钢锭",
+   
+  "item.cp_tweaks.unstable_ice_shard": "不稳定寒冰碎片",
+  
+  "item.cp_tweaks.void_wormed_sword": "虚空蠕虫剑",
+  "item.cp_tweaks.void_wormed_axe": "虚空蠕虫斧",
+  "item.cp_tweaks.void_wormed_shovel": "虚空蠕虫铲",
+  "item.cp_tweaks.void_wormed_pickaxe": "虚空蠕虫镐",
+  "item.cp_tweaks.void_wormed_hoe": "虚空蠕虫锄",
+  
+  "item.cp_tweaks.desh_sword": "戴斯剑",
+  "item.cp_tweaks.desh_axe": "戴斯斧",
+  "item.cp_tweaks.desh_shovel": "戴斯铲",
+  "item.cp_tweaks.desh_pickaxe": "戴斯镐",
+  "item.cp_tweaks.desh_hoe": "戴斯锄",
+  
+  "item.cp_tweaks.steel_sword": "钢剑",
+  "item.cp_tweaks.steel_axe": "钢斧",
+  "item.cp_tweaks.steel_shovel": "钢铲",
+  "item.cp_tweaks.steel_pickaxe": "钢镐",
+  "item.cp_tweaks.steel_hoe": "钢锄",
+  
+  "item.cp_tweaks.void_wormed_helmet": "虚空蠕虫头盔",
+  "item.cp_tweaks.void_wormed_chestplate": "虚空蠕虫胸甲",
+  "item.cp_tweaks.void_wormed_leggings": "虚空蠕虫护腿",
+  "item.cp_tweaks.void_wormed_boots": "虚空蠕虫靴",

```

```suggestion
  "item.cp_tweaks.void_wormed_boots": "虚空蠕虫靴子",
```
Minecraft原版为`靴子`
请注意全文修改

---

> Minecraft原版为`靴子` 请注意全文修改

已全文修改